### ISSUE TITLE
Bug: KeyError: 'log_file'

### ISSUE THREAD

**Issue by cgahr** [joouha/euporie#70]
Hi,

running one of
- `euporie console`
- `euporie edit`
- `euporie hub`
- `euporie notebook`
- `euporie preview`

causes the following error:

```bash
23:20:29 CRITICAL Uncaught exception                                                                                                                                   euporie.core.log.handle_exception:425                  Traceback (most recent call last):
    File "/home/<user>/.local/bin/euporie", line 8, in <module>
      sys.exit(main())
    File "/home/<user>/.local/pipx/venvs/euporie/lib/python3.10/site-packages/euporie/core/__main__.py", line 10, in main
      return entry.load().launch()
    File "/home/<user>/.local/pipx/venvs/euporie/lib/python3.10/site-packages/euporie/core/launch.py", line 29, in launch
      cls.config.load(cls)
    File "/home/<user>/.local/pipx/venvs/euporie/lib/python3.10/site-packages/euporie/core/config.py", line 374, in load
      setup_logs(self)
    File "/home/<user>/.local/pipx/venvs/euporie/lib/python3.10/site-packages/euporie/core/log.py", line 288, in setup_logs
      log_file_is_stdout = config.log_file in ("-", "/dev/stdout")
    File "/home/<user>/.local/pipx/venvs/euporie/lib/python3.10/site-packages/euporie/core/config.py", line 549, in __getattr__
      return self.get(name)
    File "/home/<user>/.local/pipx/venvs/euporie/lib/python3.10/site-packages/euporie/core/config.py", line 521, in get
      return self.settings[name].value
  KeyError: 'log_file'
```
The commands
- `euporie-console`
- `euporie-hub`
- `euporie-notebook`
- `euporie-preview`
seem to work fine.

I installed using 
```
pipx install 'euporie[hub]'
```

**Comment by joouha** [comment]
Thanks for reporting this.

This should now be fixed in the `dev` branch by a126be0fa9bd0f64d02edc57a69bbbc0507155c8.

I'll publish a new release with the fix soon.

**Comment by joouha** [comment]
This fix releases in `v2.4.0`
